docs/tutorial/electron-timelines.md

```diff
@@ -1,30 +1,96 @@
-# Electron Release Timelines
+# Electron Releases
 
-Special notes:
+Electron frequently releases major versions alongside every other Chromium release.
+This document focuses on the release cadence and version support policy.
+For a more in-depth guide on our git branches and how Electron uses semantic versions,
+check out our [Electron Versioning](./electron-versioning.md) doc.
+
+## Timeline
+
+| Electron | Alpha | Beta | Stable | Chrome | Node | Supported |
+| ------- | ----- | ------- | ------ | ------ | ---- | ---- |
+| 2.0.0 | -- | 2018-Feb-21 | 2018-May-01 | M61 | v8.9 | 🚫 |
+| 3.0.0 | -- | 2018-Jun-21 | 2018-Sep-18 | M66 | v10.2 | 🚫 |
+| 4.0.0 | -- | 2018-Oct-11 | 2018-Dec-20 | M69 | v10.11 | 🚫 |
+| 5.0.0 | -- | 2019-Jan-22 | 2019-Apr-24 | M73 | v12.0 | 🚫 |
+| 6.0.0 | -- | 2019-May-01 | 2019-Jul-30 | M76 | v12.4 | 🚫 |
+| 7.0.0 | -- | 2019-Aug-01 | 2019-Oct-22 | M78 | v12.8 | 🚫 |
+| 8.0.0 | -- | 2019-Oct-24 | 2020-Feb-04 | M80 | v12.13 | 🚫 |
+| 9.0.0 | -- | 2020-Feb-06 | 2020-May-19 | M83 | v12.14 | 🚫 |
+| 10.0.0 | -- | 2020-May-21 | 2020-Aug-25 | M85 | v12.16 | 🚫 |
+| 11.0.0 | -- | 2020-Aug-27 | 2020-Nov-17 | M87 | v12.18 | 🚫 |
+| 12.0.0 | -- | 2020-Nov-19 | 2021-Mar-02 | M89 | v14.16 | 🚫 |
+| 13.0.0 | -- | 2021-Mar-04 | 2021-May-25 | M91 | v14.16 | 🚫 |
+| 14.0.0 | -- | 2021-May-27 | 2021-Aug-31 | M93 | v14.17 | 🚫 |
+| 15.0.0 | 2021-Jul-20 | 2021-Sep-01 | 2021-Sep-21 | M94 | v16.5 | 🚫 |
+| 16.0.0 | 2021-Sep-23 | 2021-Oct-20 | 2021-Nov-16 | M96 | v16.9 | ✅ |
+| 17.0.0 | 2021-Nov-18 | 2022-Jan-06 | 2022-Feb-01 | M98 | v16.13 | ✅ |
+| 18.0.0 | 2022-Feb-03 | 2022-Mar-03 | 2022-Mar-29 | M100 | TBD | ✅ |
+| 19.0.0 | 2022-Mar-31 | 2022-Apr-30 | 2022-May-24 | M102 | TBD | ✅ |
```

Comment:
Please help me fill in this table line. I kind of winged the dates.

I also added the `needs-manual-bp` tags because the "supported" versions here will be different for each major version. I assume we want this table to be accurate once Electron 19 hits stable?

Reply:
We don't want to put E19 dates out yet. Usually we just publicize one version out (the beta one). 

E18 is in beta, which means we are supporting E15 still. E18-E15.

Reply:
The existing automation for `support.md` includes `19-x-y` as a supported release branch on `main`, so I figure it should at least be in this table, even with the dates removed.